tests/cephfs/cephfs_upgrade/cephfs_io.py

```diff
@@ -0,0 +1,69 @@
+import traceback
+from datetime import datetime, timedelta
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    try:
+        fs_util = FsUtils(ceph_cluster)
+        config = kw.get("config")
+        clients = ceph_cluster.get_ceph_objects("client")
+        build = config.get("build", config.get("rhbuild"))
+        fs_util.prepare_clients(clients, build)
+        fs_util.auth_list(clients)
+        timeout = config.get("timeout", 3600)
+        log.info("checking Pre-requisites")
+        if not clients:
+            log.info(
+                f"This test requires minimum 1 client nodes.This has only {len(clients)} clients"
+            )
+            return 1
+        client1 = clients[0]
+        stats = {"total_iterations": 0, "smallfile": 0, "dd": 0}
+        out, rc = client1.exec_command(
+            sudo=True, cmd="mount -t fuse.ceph-fuse | awk {'print $3'}", check_ec=False
+        )
```

Comment:
Can we also have IO's running on kernel mounts?

Reply:
Done